### PR TITLE
Fix/stress

### PR DESCRIPTION
**Pull request by 6l20**
## PR description

- Stress tests cleanup

## Fixed Issue(s)

Part of #852 

## Documentation

- [ ] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://github.com/consensys/orchestrate/blob/master/CHANGELOG.md).

### CODE REVIEW

File: src/api/service/types/chain.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type ChainResponse struct {
 	OwnerID                   string            `json:"ownerID,omitempty" example:"foo"`                                                       // ID of the chain owner.
 	URLs                      []string          `json:"urls" example:"https://mainnet.infura.io/v3/a73136601e6f4924a0baa4ed880b535e"`          // URLs of Ethereum nodes connected to.
 	PrivateTxManagerURL       string            `json:"privateTxManagerURL,omitempty" validate:"url"  example:"http://go-quorum/tessera:9000"` // Private tx manager required by go-quorum for sending private txs
-	ChainID                   string            `json:"chainID" example:"2445"`                                                                // [Ethereum chain ID](https://besu.hyperledger.org/en/latest/Concepts/NetworkID-And-ChainID/).
+	ChainID                   uint64            `json:"chainID" example:"2445"`                                                                // [Ethereum chain ID](https://besu.hyperledger.org/en/latest/Concepts/NetworkID-And-ChainID/).
```
Comment (ggarri) [inline src/api/service/types/chain.go:42]: `chainID` is a bigInt, therefore it must keep the string here

File: src/api/service/types/testdata/chain_registry_faker.go [hunk 2]
```diff
@@ -37,19 +33,3 @@ func FakeUpdateChainRequest() *api.UpdateChainRequest {
 	}
 }
 
-func FakeChainResponse() *api.ChainResponse {
```
Comment (ggarri) [inline src/api/service/types/testdata/chain_registry_faker.go:40]: Why did you remove this?

Comment (6l20) [reply src/api/service/types/testdata/chain_registry_faker.go:40]: It was not used, dead code

File: .circleci/config.yml [hunk 3]
```diff
@@ -454,31 +454,30 @@ workflows:
       - lint:
           filters:
             branches:
-              only: /.*/
+              ignore: main
```
Comment (ggarri) [inline .circleci/config.yml:457]: Are these changes intended?

File: src/api/service/types/chain.go (hunk 1 above)
Comment (6l20) [reply src/api/service/types/chain.go:42]: This value is given in EIP-2294 it seems, but by looking closer I did not find it has been accepted
https://github.com/ethereum/EIPs/issues/2294#:~:text=The%20maximum%20value%20of%20Chain%20ID%20is%209%2C223%2C372%2C036%2C854%2C775%2C771%20(%20MAX_CHAIN_ID%20).

Comment (ggarri) [reply src/api/service/types/chain.go:42]: I didn't know about that EIP but it seems natural to use uint64 then.

**Comment by 6l20** [comment]
This branch has been outdated by other works.
